src/components/AddBook.js

```diff
@@ -1,18 +1,40 @@
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+
+import { addBook } from '../redux/books/books';
+
 const AddBook = () => {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+
+  const dispatch = useDispatch();
+
   const handleSubmit = (e) => {
     e.preventDefault();
+
+    if (!title.trim() || !author.trim()) return;
+
+    dispatch(addBook({ title, author }));
+
```

Comment:
- [ ] as part of the requirements, you should add a unique id to every added book, so it helps you when you implement the deleting functionalities,kindly fix that 🙏 


![image](https://user-images.githubusercontent.com/89282221/207295785-7ac82b04-c9e0-46ae-8d68-ff2b48a49cbc.png)